### ISSUE TITLE
DDOptions#getOption is inefficiently parsing options

### ISSUE THREAD

**Issue by Watje** [doubledutch/DDOptions#1]
Perhaps go back to simpler parsing or cache regexp results

**Comment by kasperjj** [comment]
yeah, it's very ineffecient... but it's a tool you only use once at startup with a very limited data set. I thought about caching, but that would add the potential for leaving that cache in place for the rest of the apps runtime which seemed like a bigger performance issue.

I originally had a simpler parser, but thought this was more readable.
